src/Dialogs.vala

```diff
@@ -229,8 +229,9 @@ namespace Gala {
                 return;
             }
 
-            // Naive check to always allow inhibiting by our settings app. This is needed for setting custom shortcuts
-            if (app.id == "io.elementary.settings.desktop") {
+            if (app.id == "io.elementary.settings.desktop" || // Naive check to always allow inhibiting by our settings app. This is needed for setting custom shortcuts
```

Comment:
I wouldn't put the check here but in the center window settings, only allow centered windows for the desktop utility, deny it for the rest, then it will always fail for the `is_positioned_window` check

Reply:
Uhmm I'm not quite sure I understand what you mean 😬

Reply:
you did it ;)